app/models/user.rb

```diff
@@ -1,7 +1,7 @@
 class User < ApplicationRecord
-  # Include default devise modules. Others available are:
-  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
   devise :database_authenticatable, :registerable,
-         :recoverable, :rememberable, :validatable
+         :recoverable, :rememberable, :trackable, :validatable
+  include DeviseTokenAuth::Concerns::User
```

Comment:
In this case that's not possible because module DeviseTokenAuth::Concerns::User takes into account the devise modules we have activated in the previous line.  So if I put the include line first I get an error because Devise token auth guess that I'll use all devise options.

For more information see the following link
https://devise-token-auth.gitbook.io/devise-token-auth/usage/excluding_models#example-disable-email-confirmation